pom.xml

```diff
@@ -104,6 +104,31 @@
       <artifactId>slf4j-simple</artifactId>
       <version>1.7.25</version>
     </dependency>
+    <dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20190722</version>
+		</dependency>
+    <dependency>
+      <groupId>commons-codec</groupId>
+      <artifactId>commons-codec</artifactId>
+      <version>1.14</version>
+    </dependency>
+    <dependency>
+			<groupId>com.google.apis</groupId>
+			<artifactId>google-api-services-gmail</artifactId>
+			<version>v1-rev33-1.20.0</version>
+		</dependency>
```

Comment:
The other possibility is that if you copy and pasted code from the docs, for example, it may have used tabs instead of spaces and they didn't get automatically converted

Reply:
My cloud shell is configured to 2 spaces as tabs! The other possibility with copy/pasted dependencies is the issue; I just went ahead and updated the formatting. Good catch!